packages/deployer/.env.example

```diff
@@ -0,0 +1,2 @@
+AWS_ACCESS_KEY_ID=
```

Comment:
This file normally needs to be in the directory the container is being run from. It's also here if you want to use the deployer locally/outside the container (which you should during development, rebuilding the container takes too long).